src/pages/admin/ZendeskTicketDashboard.vue

```diff
@@ -20,13 +20,56 @@ import useEmitter from '@/hooks/useEmitter';
 import webIcon from '@/assets/icons/web.svg';
 import iosIcon from '@/assets/icons/ios.svg';
 import androidIcon from '@/assets/icons/android.svg';
+import { useToast } from 'vue-toastification';
 
 const mq = useMq();
 const { emitter } = useEmitter();
 const { t } = useI18n();
 const axiosInstance = axios.create({
   baseURL: `${import.meta.env.VITE_APP_API_BASE_URL}/zendesk`,
 });
+const toast = useToast();
+
+// Starting logic for hotline numbers
+import { formatNationalNumber } from '@/filters';
+
+const ccuApi = useApi();
+const fieldsToFetch = [
+  'id',
+  'name',
+  'short_name',
+  'active_phone_number',
+  'start_at',
+];
+const { isDataLoading, data } = ccuApi(
+  '/incidents?fields=id,name,short_name,active_phone_number&limit=200&sort=-start_at',
+  {
+    method: 'GET',
+    params: {
+      fields: fieldsToFetch.join(','),
+      limit: 200,
+      sort: '-start_at',
+    },
+  },
+);
+const incidentList = computed(() => data.value?.results);
+function getIncidentPhoneNumbers(incident) {
+  if (Array.isArray(incident.active_phone_number)) {
+    return incident.active_phone_number
+      .map((number) => formatNationalNumber(String(number)))
+      .join(', ');
+  }
+
+  return formatNationalNumber(String(incident.active_phone_number));
+}
+
+function filterNumbers(item) {
+  return item.filter((filterItem) => filterItem.active_phone_number);
+}
```

Comment:
To be refactored when by this issue: https://github.com/CrisisCleanup/crisiscleanup-4-web/issues/432